examples/interactive_play.py

```diff
@@ -279,14 +311,19 @@ def play_env(env, args, config):
         prev_time = curr_time
 
     if args.save_actions:
-        assert len(all_arm_actions) > 200
-        all_arm_actions = np.array(all_arm_actions)[:200]
-        save_dir = "orp/start_data/"
+        if len(all_arm_actions) < args.save_actions_count:
+            raise ValueError(
+                f"Only did {len(all_arm_actions)} actions but {args.save_actions_count} are required"
+            )
+        all_arm_actions = np.array(all_arm_actions)[: args.save_actions_count]
+        save_dir = "data/interactive_play_replays"
         os.makedirs(save_dir, exist_ok=True)
-        save_path = osp.join(save_dir, "bench_ac.txt")
+        save_path = osp.join(save_dir, "play_actions.txt")
```

Comment:
This should be a const string as well. Even better, it could be an optional argument so the user can manage their own recorded actions files.